pandemic/model.py

```diff
@@ -469,16 +463,22 @@ def pandemic_multiple_time_steps(
 countries = geopandas.read_file(countries_path, driver="GPKG")
 distances = distance_between(countries)
 phyto_data = pd.read_csv(phyto_path, index_col=0)
-phyto_year_cols = phyto_data.columns[3:].to_list()
-phyto_data["pc_mode"] = phyto_data[phyto_year_cols].mode(axis=1)[0]
-phyto_data.columns = np.where(
-    phyto_data.columns.isin(phyto_year_cols),
-    "Phytosanitary Capacity " + phyto_data.columns,
-    phyto_data.columns,
-)
+# Use only proactive capacity now. May incorporate reactive capacity dynamically later.
+phyto_data = phyto_data[["proactive", "ISO3", "UN"]]
+phyto_data = phyto_data.rename(columns={"proactive": "Phytosanitary Capacity"})
+
 # Assign value to phytosanitary capacity estimates
 countries = countries.merge(phyto_data, how="left", on="UN", suffixes=[None, "_y"])
-phyto_dict = {"low": phyto_low, "mid": phyto_mid, "high": phyto_high, np.nan: 0}
+phyto_dict = {
```

Comment:
Could this been done with a function to address the future enhancements that @cwaldenschreiner mentioned (e.g. give a range (0.15 to 0.90 to keep with the values here) and 0.5 to 3.0 are auto-scaled to the correct values. Makes 2 user defined parameters that can be tested during a sensitivity analysis.

Reply:
Good idea - added as an issue for future enhancement